prawns/crowdloans/src/processors/kusamaProcessor.ts

```diff
@@ -1,19 +1,18 @@
 import { SubstrateProcessor } from '@subsquid/substrate-processor';
 import crowdloanContributedHandler from '../handlers/crowdloan.contributed.event';
 import { SubstrateNetwork } from '../model';
+import { TypeormDatabase } from '@subsquid/typeorm-store'
+import { lookupArchive } from '@subsquid/archive-registry';
 
-const processor = new SubstrateProcessor('litentry_squid_crowdloans_kusama');
-
+const processor = new SubstrateProcessor(new TypeormDatabase());
 processor.setTypesBundle('kusama');
-processor.setBatchSize(500);
-processor.setIsolationLevel('REPEATABLE READ');
```

Comment:
I've been wondering the effect of removing this... as we are hoping to use the hosted service again. Although I think we will need a squid per chain though